docs/resources/deployment-examples/example-k8s/anchor-platform-dev/reference-server-deployment.yaml

```diff
@@ -20,7 +20,7 @@ spec:
     spec:
       containers:
       - name: reference-server
-        image: docker-registry.services.stellar-ops.com/dev/anchor-platform:98e5ae9
```

Comment:
I'm missing context on this; why are we moving this to dockerhub?

Reply:
dockerhub is the official repo. this docker-registry one is used by our CI/CD pipeline